contracts/erc1155/MintableERC1155.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+
+contract MintableERC1155 is ERC1155, Ownable {
+    mapping(uint256 => string) public _tokenURIs;
+
+    event Airdrop(address caller, uint256 tokenId, address[] recipients, uint256 amount);
+
+    // solhint-disable-next-line
+    constructor() ERC1155("") {}
```

Comment:
I might add a comment why you're passing in "" (because we opt to use the _tokenURIs mapping instead)